ecommerce/templates/ecommerce/review_detail.html

```diff
@@ -0,0 +1,161 @@
+              <i class="fas fa-star" style="color: orange;"></i>
+              {% for i in "1234" %}
+              <i class="fa fa-star-o"></i>
+              {% endfor %}
+            </div>
+            {% endif %}
+            <h5 class="h5 pb-2">{{ rv.title }}</h3>
+            <p class="fs-md mb-2">
+              {{ rv.content }}
+            </p>
+            {% if comments %}
+                {% for cmt in comments %}
+                <hr class="mt-4 mb-3">
+                <table id="comments">
+                <tbody>
+                  <div class="product-review pb-4">
+                      <div class="d-flex justify-content-between">
+                          <div class="d-flex align-items-center me-4 pe-2">
+                              <div style="padding-right: 20px;">
+                                  <i class="far fa-comment-dots"></i>
+                              </div>
+                              <img class ="rounded-circle" width="50" src="\media\profile_pics\avatar.png" alt="User">
+                              <div class="ps-3" style="padding-left: 20px;">
+                                  <h6 class="fs-sm mb-0">{{ cmt.user.username }}</h6>
+                                  <span class="fs-ms text-muted">{{ cmt.created }}</span>
+                              </div>
+                          </div>
+                          <div class="p-2"><i class="fas fa-reply"></i></div>
+                      </div>
+                      
+                      <div class="ps-3" style="padding-left: 105px;">
+                          <p class="fs-md mb-2">
+                              {{ cmt.comment }}
+                          </p>
+                      </div>
+                  </div>
+                </tbody>
+                </table>
+                {% endfor %}
+            {% else %}
+            <hr class="mt-4 mb-3">
+            <p class="fs-md mb-2">
+                There are no comments for this review yet.
+              </p>
+            {% endif%}
+        </div>
+        {% endfor %}
+    </div>
+</section>
+{% endblock %}
+
+{% block jscode %}
+  <script>
+    $(document).ready(function () {
+      // catch the form's submit event
+      $('#commentForm').submit(function () {
+          // create an AJAX call
+          debugger;
+          $.ajax({
+              data: $(this).serialize(), // get the form data
+              type: $(this).attr('method'), // GET or POST
+              url: $('#commentForm').attr('data-url-name'),
+              // on success
+              success: function (response) {
+                  alert(response);
+              },
+              // on error
+              error: function (response) {
+                  // alert the error if any error occured
+                  alert(response.responseJSON.errors);
+                  console.log(response.responseJSON.errors)
+              }
+          });
+          return false;
+      });
+    })
+  </script>
```

Comment:
tách ra file js riêng lưu ở folder /static nha, config static trong file settings nữa nhé
```suggestion
{% block scripts %}
  <script src="{% static 'review.js' %}" defer></script>
{% endblock scripts %}
```